docs/references/http-websocket-apis/public-api-methods/subscription-methods/subscribe.md

```diff
@@ -567,9 +704,9 @@ The `consensus` stream sends `consensusPhase` messages when [the consensus proce
 
 The fields from a consensus stream message are as follows:
 
-| `Field`             | Type                      | Description                |
-|:--------------------|:--------------------------|:---------------------------|
-| `type`              | String                    | `consensusPhase` indicates this is from the consensus stream |
-| `consensus`         | String                    | The new consensus phase the server is in. Possible values are `open`, `establish`, and `accepted`. |
+| Field       | Type   | Description                |
+|:------------|:-------|:---------------------------|
+| `type`      | String | `consensusPhase` indicates this is from the consensus stream |
```

Comment:
Perhaps "The value `consensusPhase` indicates this is from the Consensus stream." [or lowercase Book Changes stream, above]